src/main/scala/com/bsgenerator/extractor/article/HeuristicExtractor.scala

```diff
@@ -3,14 +3,13 @@ package com.bsgenerator.extractor.article
 import org.jsoup.Jsoup
 import org.jsoup.nodes.Element
 
-import scala.collection.mutable.Map
 import scala.collection.JavaConverters._
 import scala.collection.mutable
 
 object HeuristicExtractor {
 
-  val unlikelyClassCandidates: Array[String] = Array("-ad-", "ai2html", "banner", "breadcrumbs", "combx", "comment", "community", "cover-wrap", "disqus", "extra", "foot", "gdpr", "header", "legends", "menu", "related", "remark", "replies", "rss", "shoutbox", "sidebar", "skyscraper", "social", "sponsor", "supplemental", "ad-break", "agegate", "pagination", "pager", "popup", "yom-remote")
-  val likelyClassCandidates: Array[String] = Array("article", "body", "column", "main")
+  val unlikelyClassCandidates: Seq[String] = Seq("-ad-", "ai2html", "banner", "breadcrumbs", "combx", "comment", "community", "cover-wrap", "disqus", "extra", "foot", "gdpr", "header", "legends", "menu", "related", "remark", "replies", "rss", "shoutbox", "sidebar", "skyscraper", "social", "sponsor", "supplemental", "ad-break", "agegate", "pagination", "pager", "popup", "yom-remote")
```

Comment:
Surprise! Default Scala Seq is mutable unless you import it from `collection.immutable`
https://orrsella.com/2014/02/27/the-default-scala-seq-is-mutable-wait-what/

Reply:
That's actually not true. The default Seq builder returns an immutable list.

collection.Seq:
`def newBuilder[A]: Builder[A, Seq[A]] = immutable.Seq.newBuilder[A]` 
where mutable.ListBuffer's toResult yields immutable.List

The article states that if we were to accept collection.Seq somewhere it could be both mutable or immutable implementation.